distribution/lib/Standard/Base/0.0.0-dev/src/Data/Download/Download_Mode.enso

```diff
@@ -1,8 +1,7 @@
 import project.Data.Time.Date_Time.Date_Time
 import project.Data.Time.Duration.Duration
-import project.Nothing.Nothing
 import project.System.File.Generic.Writable_File.Writable_File
-from project.Data.Boolean import Boolean, False, True
```

Comment:
The `Boolean` imports were being added like this deliberately - the reason was it seemed weird to have `True` in scope but not `False` - in case the code is modified it may be surprising.

Ofc it's just our internals so ok to keep like that and we can add import if needed.

But perhaps we should change this case to
```
from project.Data.Boolean import all
```

@jdunkerley thoughts?

Reply:
Outdated, since the changes to Enso code will be done in subsequent PRs. See https://github.com/enso-org/enso/pull/13017#issuecomment-2871791447.

But if you would change the import to `from project.Data.Booleam import all`, it will be ignored by the compiler pass.